docs/code-reviews/recipes/csharp.md

```diff
@@ -47,11 +47,11 @@ The [.editorconfig](https://docs.microsoft.com/en-us/visualstudio/ide/editorconf
 
 [Details about the configuration of different rules](https://docs.microsoft.com/en-us/visualstudio/code-quality/use-roslyn-analyzers?view=vs-2019).
 
-### FxCop analyzers
+### .NET analyzers
 
-Microsoft's FxCop analyzers check your code for security, performance, and design issues, among others. [Install FxCop analyzers in Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers?view=vs-2019).
+Microsoft's .NET analyzers has code quality rules and .NET API usage rules implemented as analyzers using the .NET Compiler Platform (Roslyn). This is the replacement for Microsoft's legacy [FxCop analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/). Starting in Visual Studio 2019 16.8 and .NET 5.0, these analyzers are included with the .NET SDK. It is recommended that you enable the analyzers from the .NET SDK when possible. [Enable or install first-party .NET analyzers](https://docs.microsoft.com/en-us/visualstudio/code-quality/install-net-analyzers?view=vs-2019).
```

Comment:
I think this is duplicate from the link you are sharing. I would not include that. 

> Starting in Visual Studio 2019 16.8 and .NET 5.0, these analyzers are included with the .NET SDK. It is recommended that you enable the analyzers from the .NET SDK when possible.

Reply:
Removed any duplicate information.